genie_registry/clinical.py

```diff
@@ -823,6 +843,12 @@ def _validate(self, clinicaldf):
         )
         total_error.write(error)
 
+        # CHECK: YEAR DEATH against YEAR CONTACT
+        haveColumn = process_functions.checkColExist(clinicaldf, ["YEAR_DEATH", "YEAR_CONTACT"])
```

Comment:
I don't think `process.functions.checkColExist` accepts a list of keys. It would have to be an individual column.key you use.

Reply:
I tested it when I wrote this. The `checkColExist` output a dataframe with the filled column names and if the output dataframe is not empty it returns True otherwise False. This works when we want to check if both `YEAR_DEATH` and `YEAR_CONTACT` exist concurrently. Here are test cases I used:
<img width="699" alt="Screenshot 2024-04-22 at 1 36 13 PM" src="https://github.com/Sage-Bionetworks/Genie/assets/90745557/0f9abd5b-72e5-4d64-a31a-dcb209c3d52c">

Reply:
Neat! Could you update the docstring of `process_functions.checkColExist` and any tests (to have multiple column checks as test cases if not already) to reflect this? Thanks!

Reply:
Sure.